Gemfile

```diff
@@ -80,3 +90,18 @@ end
 group(:linux, :bsd, :mac_os_x, :solaris) do
   gem "ruby-shadow"
 end
+
+# TODO delete this when we figure out how to include the pushy windows dependencies
+# correctly
+platforms :mswin, :mingw do
```

Comment:
Also - if pushy-client doesn't need anything other than the dependencies that chef-client needs, it should just transitively use chef-client's dependencies.  Those windows deps there are actually extremely...  fragile.  See https://github.com/chef/chef/blob/master/chef-universal-mingw32.gemspec#L12 for an example.  There are a couple other booby-traps waiting for the unwary adventurer.  Best to consolidate all the windows specific magic in one place.